### ISSUE TITLE
404Not found

### ISSUE THREAD

**Issue by Trigun** [causefx/Organizr#515]
Hi, Kinda new to this but looking forward of using it!
I am using a Synology, installed NginX, php7, followed the wiki, copied the files to a folder in the "web"rootfolder, in folder "Organzr" but I get the 404 msg.

can you please help me out here?

thnx!

**Comment by MrMxyzptlk** [comment]
Do yourself a favor and install Docker on your Synology and then the Organizr image.
It takes all of 5 minutes, with almost 0 configuration. 

All I configured was I changed the port settings.

**Comment by Trigun** [comment]
Cheers!! Sounds good! Will do so.

Kind regards/Met vriendelijke
Groet,

Bas Wiggers

Sent from my iPhone

On 21 Jul 2017, at 17:30, MrMxyzptlk <notifications@github.com<mailto:notifications@github.com>> wrote:


Do yourself a favor and install Docker on your Synology and then the Organizr image.
It takes all of 5 minutes, with almost 0 configuration.

All I configured was I changed the port settings.

**Comment by Trigun** [comment]
Damn, no docker package available for my ds215+
Any other options perhaps?

Kind regards/Met vriendelijke
Groet,

Bas Wiggers

Sent from my iPhone

On 21 Jul 2017, at 17:30, MrMxyzptlk <notifications@github.com<mailto:notifications@github.com>> wrote:


Do yourself a favor and install Docker on your Synology and then the Organizr image.
It takes all of 5 minutes, with almost 0 configuration.

All I configured was I changed the port settings.
